puppeteerInterface.js

```diff
@@ -158,10 +158,10 @@ class PuppeteerInterface {
         await this.ready
         if (!this.backgroundWorker) return []
         return await this.backgroundWorker.evaluate(
-            async testRequest =>
+            async requestDetails =>
```

Comment:
Nit: Please could you rename the testMatchOutcome parameter to `testRequestDetails` in this case?